packages/domReady/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@wordpress/domReady",
```

Comment:
We'll need this to be dash-delimited:

>New packages must not have uppercase letters in the name.

https://docs.npmjs.com/files/package.json#name